apps/web/components/examples/blockchain/graduation-dashboard.tsx

```
@@ -0,0 +1,389 @@
+        {/* Badge Collection */}
+        <div className="bg-white rounded-lg border border-gray-200 p-6">
+          <h2 className="text-lg font-semibold text-gray-900 mb-4 flex items-center gap-2">
+            <TrophyIcon className="h-5 w-5" />
+            Badge Collection
+          </h2>
+          
+          {badges.length > 0 ? (
+            <div className="grid grid-cols-3 gap-3">
+              {badges.slice(0, 6).map((badge, index) => (
+                <div 
+                  key={index}
```

Biome (2.1.2)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)